DESCRIPTION

```diff
@@ -1,15 +1,52 @@
 Package: geoGraph
-Version: 1.0-0
-Date: 2015/09/13
-Title: walking through the geographic space using graphs
-Author: Thibaut Jombart
+Type: Package
+Title: What the Package Does (Title Case)
+Version: 1.0-1
+Author: Thibaut Jombart,
+  Andrea Manica
 Maintainer: Thibaut Jombart <t.jombart@imperial.ac.uk>
```

Comment:
Replace with Andrea. Should be simple as the package is not on CRAN.

Reply:
Done.